magprime/templates/regextra.html

```diff
@@ -24,15 +24,18 @@
             $('#guitar_keychain,#tax_exempt').empty();
         }
         {% if c.PAGE_PATH != '/registration/form' %}
+            // TODO: rip this hardcoded stuff out?
             if ($.field('amount_extra')) {
                 $.field('amount_extra').parents('.form-group').append($('#tax_exempt'));
                 $("input:radio[name='amount_extra'][value=40]").parents('.btn').append($('#guitar_keychain'));
             }
+
             if ($.field('shirt')) {
                 $.field('shirt').parents('.form-group').append($('#tshirt_warning'));
             } else {
                 $('#tshirt_warning').hide();
             }
+            // TODO: rip all of this hardcoded swag and pricing out?
```

Comment:
Yes, kill it with a lot of fire. This was all just a super-last-minute "oh god we have to slap preview links on some of the kick-in items, oh god oh god."

The pricing isn't hardcoded, by the way, the 'values' you see here are to select _which buttons_ to add things to.